pkg/catalog/catalog.go

```diff
@@ -179,12 +180,15 @@ func createImageTagMap(tagRefs []imagev1.TagReference) map[string]string {
 // https://github.com/openshift/odo-init-image/blob/master/language-scripts/image-mappings.json
 func isSupportedImage(imgName string) bool {
 	supportedImages := []string{
-		"redhat-openjdk-18/openjdk18-openshift",
-		"openjdk/openjdk-11-rhel8",
-		"openjdk/openjdk-11-rhel7",
-		"rhscl/nodejs-8-rhel7",
-		"rhoar-nodejs/nodejs-8",
-		"rhoar-nodejs/nodejs-10",
+		"redhat-openjdk-18/openjdk18-openshift:latest",
+		"openjdk/openjdk-11-rhel8:latest",
+		"openjdk/openjdk-11-rhel7:latest",
+		"rhscl/nodejs-8-rhel7:latest",
+		"rhoar-nodejs/nodejs-8:latest",
+		"rhoar-nodejs/nodejs-10:latest",
+		"bucharestgold/centos7-s2i-nodejs:8.x",
```

Comment:
does `.x` work correctly, like... is it actually tagged `.x` or is `.x` a filler for "whatever number"

Reply:
It is actually a tag